neuralprophet/time_net.py

```diff
@@ -794,19 +796,25 @@ def training_step(self, batch, batch_idx):
         optimizer.step()
 
         scheduler = self.lr_schedulers()
-        scheduler.step()
+        if self.finding_lr:
+            scheduler.step()
+        else:
+            scheduler.step(epoch=epoch_float)
 
-        # Manually track the loss for the lr finder
-        self.log("train_loss", loss, on_step=False, on_epoch=True, prog_bar=True, logger=True)
-        self.log("reg_loss", reg_loss, on_step=False, on_epoch=True, prog_bar=True, logger=True)
+        if self.finding_lr:
+            # Manually track the loss for the lr finder
+            self.log("train_loss", loss, on_step=False, on_epoch=True, prog_bar=True, logger=True)
```

Comment:
pass log_args

Reply:
check for lr-finder requirements

Reply:
implement better prgress/metrics logging/printing. 
e.g. also need to touch `time_net.__init__`
```
self.log_args = {
                "on_step": False,
                "on_epoch": True,
                "prog_bar": True,
                "batch_size": self.config_train.batch_size,
            }
```